Bundle mocha with webpack to extend functionality

### Prerequisites

- [X] Checked that your issue hasn't already been filed by cross-referencing [issues with the `faq` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Afaq%20)
- [X] Checked next-gen ES issues and syntax problems by using the same environment and/or transpiler configuration without Mocha to ensure it isn't just a feature that actually isn't supported in the environment in question or a bug in your code.
- [X] 'Smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, your usage of Mocha, or Mocha itself
- [X] Ensured that there is no discrepancy between the locally and globally installed versions of Mocha. You can find them with: `node node_modules/.bin/mocha --version`(Local) and `mocha --version`(Global). We recommend that you _not_ install Mocha globally.

### Description

I am working on Angular 2 with Cypress. I would like to extend mocha's functionality to add a custom metadata object inside it() and describe(). To do this, i obviously need to require mocha. Code: https://github.com/devhazem/mocha-test-options/blob/master/index.js 

When doing this, I get the following error (even though the file does exist):
`Error: Webpack Compilation Error
C:/app/node_modules/mocha/lib/browser/growl.js
Module not found: Error: Can't resolve '../../package' in 'C:\app\node_modules\mocha\lib\browser'
resolve '../../package' in 'C:\app\node_modules\mocha\lib\browser'
  using description file: C:\app\node_modules\mocha\package.json (relative path: ./lib/browser)
    using description file: C:\app\node_modules\mocha\package.json (relative path: ./package)
`
How can I achieve this?

### Steps to Reproduce

- Use https://github.com/devhazem/mocha-test-options as a dev dependency in any Angular 2 project
- declare module:
`declare module mocha-test-options:
  export function describe(title: string, metadata: object, fn);
  export function it(title: string, metadata: object, fn);`
- import declared module

**Expected behavior:** should not throw any errors

**Actual behavior:** 
`Error: Webpack Compilation Error
C:/app/node_modules/mocha/lib/browser/growl.js
Module not found: Error: Can't resolve '../../package' in 'C:\app\node_modules\mocha\lib\browser'
resolve '../../package' in 'C:\app\node_modules\mocha\lib\browser'
  using description file: C:\app\node_modules\mocha\package.json (relative path: ./lib/browser)
    using description file: C:\app\node_modules\mocha\package.json (relative path: ./package)`

**Reproduces how often:** All the time

### Versions

mocha: 8.2.0
typescript: "3.9.4"

--- a/lib/browser/growl.js
+++ b/lib/browser/growl.js
@@ -125,7 +125,7 @@ function display(runner) {
     cross: '\u274C',
     tick: '\u2705'
   };
-  var logo = require('../../package').notifyLogo;
+  var logo = require('../../package.json').notifyLogo;
   var _message;
   var message;
   var title;

